admin-client/src/main/java/com/epam/admin_client/config/SecurityConfig.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.admin_client.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@EnableWebSecurity
+@ComponentScan("com.epam.admin_client")
```

Comment:
This annotation is inferred from "@SpringBootApplication", therefore not required at all